keeper-gradle-plugin/src/main/java/com/slack/keeper/InferAndroidTestKeepRules.kt

```diff
@@ -63,13 +66,21 @@ public abstract class InferAndroidTestKeepRules : JavaExec() {
   @get:Input
   public abstract val jvmArgsProperty: ListProperty<String>
 
+  /** @see KeeperExtension.keepRulesArgs */
+  @get:Input
+  public abstract val keepRulesArgs: ListProperty<String>
+
   /**
    * Enable more descriptive precondition checks in the CLI. If disabled, errors will be emitted to
    * the generated proguard rules file instead.
    */
   @get:Input
   public abstract val enableAssertionsProperty: Property<Boolean>
 
+  /** @see KeeperExtension.enableTraceReferences */
+  @get:Input
+  public abstract val enableTraceReferences: Property<Boolean>
```

Comment:
@ZacSweers I tried to create a similar syntax than what you mentioned but it was not as easy.
I found a [similar idea in the Gradle Slack](https://gradle-community.slack.com/archives/CA745PZHN/p1599216624018900) but the solution seems not really clean compare to just expose the enable flag like that.

If you have a better idea about how to implement it, please let me know 🙏